integration/scripts/proxyServer.ts

```diff
@@ -26,7 +26,7 @@ export const createProxyServer = (opts: ProxyServerOptions) => {
       res.writeHead(404);
       res.end();
     }
-  }).listen(usingSSL ? 443 : 80);
+  }).listen(usingSSL ? 443 : 80, '127.0.0.1');
```

Comment:
Blacksmith's network configuration needs us to explicitly bind to `127.0.0.1` instead of the default.